.vault-config/product-construction-int.yaml

```diff
@@ -4,28 +4,7 @@ storageLocation:
     subscription: e6b5f9f5-0ca4-4351-879b-014d78400ec2
     name: ProductConstructionInt
 
-references:
-  helixkv:
-    type: azure-key-vault
-    parameters:
-      subscription: a4fc5514-21a9-4296-bfaf-5c7ee7fa35d1
-      name: helixkv
-  
-  engkeyvault:
-    type: azure-key-vault
-    parameters:
-      subscription: a4fc5514-21a9-4296-bfaf-5c7ee7fa35d1
-      name: engkeyvault
-
 secrets:
-  BotAccount-dotnet-bot-repo-PAT:
```

Comment:
Isn't this used for the GitHub release? Maybe we should rather keep it in the prod PCS manifest so that we don't lose it when we delete the Maestro manifests?

Reply:
Ah, It is, I didn't notice it when I was looking at the secret usage, probably looked at `.cs` files only. Yeah I think we only need it in the prod manifest